userproject/Model_Schmuker_2014_classifier_project/README.txt

```diff
@@ -1,66 +1,62 @@
-Author: Alan Diamond, University of Sussex, 2014
```

Comment:
Do we need to add the author as it creates a bit of confusion...also it has already been cited in the model files